gitops/git/github/github.go

```diff
@@ -34,6 +35,16 @@ func CreatePR(from, to, title string) error {
 	)
 	tc := oauth2.NewClient(ctx, ts)
 	gh := github.NewClient(tc)
```

Comment:
I'm not sure what are the possible side effects from the `github.NewClient()`, but I'd avoid creating the instance of this client in the Github Enterprise case.

Reply:
Good suggestion. I will make this an an exclusive or client creation.